modules/debts_api/app/sidekiq/debts_api/v0/form5655/vba_submission_job.rb

```diff
@@ -8,21 +8,24 @@ class V0::Form5655::VBASubmissionJob
     include SentryLogging
     STATS_KEY = 'api.vba_submission'
 
-    sidekiq_options retry: false
+    sidekiq_options retry: 5, retry_interval: ->(_) { 1.hour.to_i }
```

Comment:
I'm interested to hear if you(the reviewer) have better ideas than what I've done here :).

Reply:
@digitaldrk Would `sidekiq_retry_in` be better? I see it used a few times in vets-api.
This is the most documentation I could find: https://github.com/sidekiq/sidekiq/blob/main/lib/sidekiq/job_retry.rb#L203-L205